lib/mayaUsd/commands/Readme.md

```diff
@@ -0,0 +1,440 @@
+| `-stripNamespaces`               | `-sn`      | bool             | false               | Remove namespaces during export. By default, namespaces are exported to the USD file in the following format: nameSpaceExample_pPlatonic1 |
+| `-staticSingleSample`            | `-sss`     | bool             | false               | Converts animated values with a single time sample to be static instead |
+| `-verbose`                       | `-v`       | noarg            | false               | Make the command output more verbose |
+
+#### Frame Samples
+
+The frame samples are computed *after* the frame stride is taken
+into account. If any of your frame samples falls outside the open
+interval `(-frameStride, +frameStride)`, a warning will be issued,
+but export will proceed normally.
+
+**Note**: If you have frame samples > 0.0, additional frames may
+          be generated outside your frame range.
+
+Examples:
+
+| frameRange   | frameStride | frameSample    | Time samples in exported USD file                |
+| ------------ | ----------- | -------------- | ------------------------------------------------ |
+| [1, 3]       | 1.0         | -0.1, 0.2      | 0.9, 1.2, 1.9, 2.2, 2.9, 3.2                     |
+| [1, 3]       | 1.0         | -0.1, 0.0, 0.2 | 0.9, 1.0, 1.2, 1.9, 2.0, 2.2, 2.9, 3.0, 3.2      |
+| [1, 3]       | 2.0         | -0.1, 0.2      | 0.9, 1.2, 2.9, 3.2                               |
+| [1, 3]       | 0.5         | -0.1, 0.2      | 0.9, 1.2, 1.4, 1.7, 1.9, 2.2, 2.4, 2.7, 2.9, 3.2 |
+| [1, 3]       | 2.0         | -3.0, 3.0      | -2.0, -1.0, 4.0, 5.0                             |
+
+The last example is quite strange, and a warning will be issued.
+This is how it will be processed:
+
+* The current time starts at 1.0. We evaluate frame samples,
+  giving us time samples -2.0 and 4.0.
+* Then we advance the current time by the stride (2.0),
+  making the new current time 3.0. We evaluate frame samples,
+  giving us time samples -1.0 and 5.0.
+* The time samples are sorted before exporting, so they are
+  evaluated in the order -2.0, -1.0, 4.0, 5.0.
+
+
+### Export Behaviors
+
+#### Model Hierarchy and Kind
+
+`MayaUSDExportCommand` currently attempts to make each root prim in the
+exported USD file a valid model, and may author a computed `kind`
+on one or more prims to do so. In the future, we plan to support
+annotating desired `kind` in the Maya scenegraph, and possibly make
+further `kind`/model inference optional. The current behavior is:
+
+* We initially author the kinds on prims based on the `-kind` flag
+  in `MayaUSDExportCommand` (see the `MayaUSDExportCommand` flags above) or the
+  `USD_kind` attribute on individual Maya nodes (see the "Maya
+  Custom Attributes" table below).
+
+* For each root prim in the scene, we *validate* the kind if it
+  has been specified. Otherwise, we *compute* a kind for the root prim:
+  * If the root prim is specified to be an `assembly` (or type derived
```

Comment:
I did not touch this section. Here there is a mention of assembly, which I was not sure of its correctness.

Reply:
This should all still be relevant, since "assembly" here refers to the USD kind and **not** to Maya's MFnAssembly.